app/web/features/auth/notifications/PushNotificationPermission.tsx

```diff
@@ -0,0 +1,236 @@
+import { Typography } from "@material-ui/core";
```

Comment:
makes sense to create two components?

- one would be the frontside component (used in the feature preview)
- other one for the internal logic, which would be rendered (I guess) in every page, like wrapping the app, or rendered in the footer (or wherever)

Reply:
Why would we want the logic on every page? You might be thinking of toast notification, which would run within the app (and thus make sense wrapping the app with a NotificationContext or something). Since these are system notifications, they are handled in the `service-worker.js` file totally outside of our web app.

I think might be an over-optimization at this point since we don't know where we will move it yet, or what we will use to trigger the permission request. IMO probably makes sense we move the buttons to the new notification settings page?

Reply:
yeah makes sense...

about moving it to the notification settings page isn't up to me, so I can't decide but makes sense though, I suggest to keep it in the feature preview page for now and move it there once we are ready